packages/radix-ui-themes/package.json

```diff
@@ -84,41 +84,41 @@
     "prepublishOnly": "pnpm lint"
   },
   "dependencies": {
-    "@radix-ui/colors": "3.0.0",
-    "@radix-ui/primitive": "1.1.0",
-    "@radix-ui/react-accessible-icon": "1.1.0",
-    "@radix-ui/react-alert-dialog": "1.1.1",
-    "@radix-ui/react-aspect-ratio": "1.1.0",
-    "@radix-ui/react-avatar": "1.1.0",
-    "@radix-ui/react-checkbox": "1.1.0",
-    "@radix-ui/react-compose-refs": "1.1.0",
-    "@radix-ui/react-context": "1.1.0",
-    "@radix-ui/react-context-menu": "2.2.1",
-    "@radix-ui/react-dialog": "1.1.1",
-    "@radix-ui/react-direction": "1.1.0",
-    "@radix-ui/react-dropdown-menu": "2.1.1",
+    "@radix-ui/colors": "^3.0.0",
+    "@radix-ui/primitive": "^1.1.0",
+    "@radix-ui/react-accessible-icon": "^1.1.0",
+    "@radix-ui/react-alert-dialog": "^1.1.2",
+    "@radix-ui/react-aspect-ratio": "^1.1.0",
+    "@radix-ui/react-avatar": "^1.1.1",
+    "@radix-ui/react-checkbox": "^1.1.2",
+    "@radix-ui/react-compose-refs": "^1.1.0",
+    "@radix-ui/react-context": "^1.1.1",
+    "@radix-ui/react-context-menu": "^2.2.2",
+    "@radix-ui/react-dialog": "^1.1.2",
+    "@radix-ui/react-direction": "^1.1.0",
+    "@radix-ui/react-dropdown-menu": "^2.1.2",
     "@radix-ui/react-form": "0.1.0",
```

Comment:
Left this one pinned since it's not a stable release. We may be able to go ahead and ship a 1.0 since I haven't seen any major issues with the API, and we haven't planned anything around this AFAIK.

Reply:
I didn't even know this was a dependency 🧐

Reply:
Hah, we aren't even using it. I'll just remove!